sklearnex/spmd/basic_statistics/tests/test_basic_statistics_spmd.py

```diff
@@ -18,7 +18,7 @@
 import pytest
 from numpy.testing import assert_allclose
 
-from onedal.tests.utils._spmd_support import (
+from sklearnex.tests._utils_spmd import (
```

Comment:
Better to use relative imports inside module. Please fix it for the rest of the files here

Reply:
fixed

Reply:
Yields `ImportError: attempted relative import with no known parent package`. Returning to non relative import for now, unless there is a suggested workaround that would allow for relative import

Reply:
due to sklearn guideline imports must be absolute in tests, so no need to look for workaround
https://scikit-learn.org/stable/developers/develop.html